packages/components-js/projects/jsdom-polyfill/rollup.config.js

```diff
@@ -24,23 +24,19 @@ export default [
         find: /'ROLLUP_REPLACE_VERSION'/,
         replace: `'${version}'`,
       }),
-      modify({
-        find: /console.warn\(`The Porsche Design System had to inject our font-face\.css file into your head\.[\s\S]*?`\);/,
-        replace: '',
-      }),
       // patch conditions into build to allow opt out of CDN requests
       modify({
         // font-face css
         find: /appGlobals\.globalScripts\(\);/,
         replace: (match) => `if(!window.PDS_SKIP_FETCH) { ${match} }`,
       }),
       modify({
-        // icon svgs
-        find: /(const pdsFetch = \(input, init\) =>) (fetch\(input, init\);)/,
-        replace: (_, $1, $2) => `${$1} window.PDS_SKIP_FETCH ? Promise.resolve({ ok:true, text: () => ''}) : ${$2}`,
+        // icon svgs (img src)
+        find: /(src:) (buildIconUrl\()/,
+        replace: (_, $1, $2) => `${$1} window.PDS_SKIP_FETCH ? undefined : ${$2}`,
```

Comment:
The global variable `PDS_SKIP_FETCH` is still needed, right?

Reply:
Yes, but maybe not fully working everywhere since newer components were not adjusted.